{"name": "test_multiple_login_page_success[invalid_user2-wrong_password]", "status": "passed", "parameters": [{"name": "username", "value": "'invalid_user2'"}, {"name": "password", "value": "'wrong_password'"}], "start": 1753503455186, "stop": 1753503457345, "uuid": "e6e30413-9574-485f-9fef-bbe4f3d97ca7", "historyId": "bff793983932d93056520584ccf3f372", "testCaseId": "90d4f72dd78db4ae114ce1f1cc9587a0", "fullName": "tests.test_multiple_login#test_multiple_login_page_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_multiple_login"}, {"name": "host", "value": "ahmads-MacBook-Pro.local"}, {"name": "thread", "value": "58931-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_multiple_login"}], "titlePath": ["tests", "test_multiple_login.py"]}